modules/01 JavaScript Programming/Assignments/25/Answer

```diff
@@ -0,0 +1,35 @@
+(function(){
```

Comment:
good solution. 2 comments:
1. pay attention to your mistakes in lines 25 and 28. your interviewer might think that you solved it incorrectly because the output is wrong
2. format your code before submitting. the cases code doesn't look good